### PR TITLE
Set url/token with policy

### PR DESCRIPTION
**Pull request by andvra**
---
Please check the following conditions for your PR.

- [ ] `apiDefinition.swagger.json` is validated using `paconn validate` command.
- [ ] `apiProperties.json` has a valid brand color. Invalid brand colors are `#007ee5` and `#ffffff`.

### CODE REVIEW

File: certified-connectors/Peltarion/apiDefinition.swagger.json [hunk 1]
```diff
@@ -47,34 +47,6 @@
...
-            "in": "header",
-            "required": true,
-            "type": "string",
-            "x-ms-visibility": "important",
-            "description": "Peltarion token",
-            "x-ms-summary": "Token to identify your Peltarion API"
-          },
           {
             "name": "peltarionbody",
             "in": "header",
```
Comment (sriyen-msft) [inline certified-connectors/Peltarion/apiDefinition.swagger.json:52]: @andvra The summary and description says it is body parameter, but definition says "in":"header". If it is body parameter that expects json object, it should have "type":"object".

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Peltarion/apiDefinition.swagger.json (hunk 1 above)
Comment (andvra) [reply certified-connectors/Peltarion/apiDefinition.swagger.json:52]: Our platform expects body tags with a number of different structures, so it won't match a specific schema, which is required if we send this as a body. Therefore, we stringify the body and send it as a header parameter.

File: certified-connectors/Peltarion/apiProperties.json [hunk 2]
```diff
@@ -13,10 +36,36 @@
...
-          ]
+          "x-ms-apimTemplate-policySection": "Request"
+        }
+      },
+      {
+        "templateId": "setqueryparameter",
+        "title": "Query token",
+        "parameters": {
+          "x-ms-apimTemplateParameter.name": "code",
+          "x-ms-apimTemplateParameter.value": "FHWJvOJnrrBz8v7DOWoABLS1K2e9Im6q7YalAthQMGkqTa57jDYm/w==",
```
Comment (sriyen-msft) [inline certified-connectors/Peltarion/apiProperties.json:47]: @andvra It seems to be some shared key to access your service. 
1) Please note that this key is publicly visible to anyone.
2) You may place into connection parameters (as url and token) and share the key with your customers explicitly

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/Peltarion/apiProperties.json (hunk 2 above)
Comment (andvra) [reply certified-connectors/Peltarion/apiProperties.json]: Thanks for noting. The code isn't needed, so removed it from the code now.

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**